packages/otp2-tile-overlay/src/index.tsx

```diff
@@ -135,6 +141,9 @@ const OTP2TileLayerWithPopup = ({
   if (type === "stops" || type === "areaStops") {
     filter = ["all", ["!", ["has", "parentStation"]], ["!=", ["get", "routes"], ["literal", "[]"]]]
   }
+  if (stopsWhitelist) {
+    filter = ["in", ["get", "gtfsId"], ["literal", stopsWhitelist]]
+  }
```

Comment:
will it ever be a problem if this is overriding the previous filters set or is this intentional?

Reply:
We want to disable the old ones. We want to show all stops even if there is a parent station! And showing stops without routes is impossible (since we are generating the whitelist from route data) but even if not I like that the whitelist overrides existing filters